### PR TITLE
Update ideas query to retrieve ideas in config order.

### PR DESCRIPTION
**Pull request by microbit-robert**

### CODE REVIEW

File: src/documentation/ideas/content.ts [hunk 1]
```diff
@@ -14,25 +14,29 @@ const ideasQuery = (languageId: string): string => {
...
     }`;
 };
 
 const adaptContent = (result: any): Idea[] | undefined => {
-  const ideas = result as Idea[];
-  if (ideas.length === 0) {
-    return undefined;
+  const ideas = result?.pythonIdeasOrder as Idea[];
+  if (ideas) {
+    if (ideas.length === 0) {
```
Comment (microbit-matt-hillsdon) [inline src/documentation/ideas/content.ts:37]: This looks slightly odd.

**Comment by microbit-matt-hillsdon** [comment]
Confirmed fr to en fallback is working. Some French ideas will be lost until the next translation sync though we're planning to run an extra one today.

**Review by microbit-matt-hillsdon — APPROVED** [review]
Query looks good. One minor comment above.

File: src/documentation/ideas/content.ts (hunk 1 above)
Comment (microbit-robert) [reply src/documentation/ideas/content.ts]: More logical check here -> [`80ad3b2`](https://github.com/microbit-foundation/python-editor-next/pull/837/commits/80ad3b25970c049196b391afded3b030e5e20292).

**Pull request merged**